docs/guides/real-time/operations.md

```diff
@@ -0,0 +1,152 @@
+---
+description: "Learn how to execute CRUD operations over Directus' WebSockets interface"
+contributors: Kevin Lewis
+---
+
+# WebSockets Operations
+
+You can execute CRUD operations over Directus' WebSockets interface.
+
+This guide assumes you already know [how to establish, authenticate, and send messages](/guides/real-time/getting-started/websockets) over a WebSocket connection.
+
+:::info GraphQL
+
+The GraphQL Subscriptions specification does not support CRUD operations. This guide is only suitable for WebSockets connections not using GraphQL.
+
+:::
+
+## Read Items
+
+```json
+{
+  "type": "items",
+  "action": "read",
+  "collection": "collection_name",
```

Comment:
I think users may mistakenly assume that `collection_name` means inputting `messages_name` for a messages collection. To avoid confusion, it could be helpful to add a note clarifying that `collection_name` simply refers to the name of a collection in their Directus project. Same for `item_id`